lib/src/components/NxBinaryDonutChart/__tests__/NxDonutChart.test.tsx

```diff
@@ -4,304 +4,224 @@
     });
 
-    it('renders only the background path with the expected properties at zero value', function() {
-      const circles = getShallowComponent().find('circle');
-      const paths = getShallowComponent().find('path');
-
-      expect(circles.length).toBe(0);
-      expect(paths.length).toBe(1);
-      expect(paths.at(0)).toHaveClassName('.nx-binary-donut-chart__background');
-
-      // whether the background renders correctly is best left to the visual tests
+    it('does not render an arc when value is zero', function() {
+      expect(renderArc()).not.toBeInTheDocument();
     });
 
-    it('renders only the background path with the expected properties at zero value with maxVal',
-        function() {
-          const circles = getShallowComponent({maxVal: 200}).find('circle');
-          const paths = getShallowComponent({maxVal: 200}).find('path');
-
-          expect(circles.length).toBe(0);
-          expect(paths.length).toBe(1);
-          expect(paths.at(0)).toHaveClassName('.nx-binary-donut-chart__background');
-
-        // whether the background renders correctly is best left to the visual tests
-        });
-
-    it('renders the background path and an arc circle with the expected properties when value is 100',
-        function() {
-          const circles = getShallowComponent({value: 100}).find('circle');
-          const paths = getShallowComponent({value: 100}).find('path');
-
-          expect(circles.length).toBe(1);
-          expect(circles.at(0)).toHaveClassName('.nx-binary-donut-chart__arc');
-          expect(circles.at(0)).toHaveProp('strokeWidth', 7);
-          expect(circles.at(0)).toHaveProp('r', 10.5);
-          expect(paths.length).toBe(1);
-          expect(paths.at(0)).toHaveClassName('.nx-binary-donut-chart__background');
-        });
-
-    it('renders the background path and an arc circle with the expected properties when value is equal to maxVal',
-        function() {
-          const circles = getShallowComponent({value: 80, maxVal: 80}).find('circle');
-          const paths = getShallowComponent({value: 80, maxVal: 80}).find('path');
-
-          expect(circles.length).toBe(1);
-          expect(circles.at(0)).toHaveClassName('.nx-binary-donut-chart__arc');
-          expect(circles.at(0)).toHaveProp('strokeWidth', 7);
-          expect(circles.at(0)).toHaveProp('r', 10.5);
-          expect(paths.length).toBe(1);
-          expect(paths.at(0)).toHaveClassName('.nx-binary-donut-chart__background');
-        });
-
-    it('renders only a background circle path when value is negative', function() {
-      const circles = getShallowComponent({value: -50}).find('circle');
-      const paths = getShallowComponent({value: -50}).find('path');
-
-      expect(circles.length).toBe(0);
-      expect(paths.length).toBe(1);
-      expect(paths.at(0)).toHaveClassName('.nx-binary-donut-chart__background');
+    it('renders a correct arc path when 0 < value < 100', function() {
```

Comment:
Just curious if we should have a test when the value is above maxVal?

Reply:
> Just curious if we should have a test when the value is above maxVal?

Yeah, and even more importantly one for when value is _at_ `maxVal`.  Granted, in either case there's a limited amount that it makes sense to test without visuals.  But nonetheless, [fe7675a](https://github.com/sonatype/sonatype-react-shared-components/pull/930/commits/fe7675a26bc30dcec2fc09117a6708ac3c60bcb0)